shared/src/androidMain/kotlin/di/PlatformModules.android.kt

```diff
@@ -3,13 +3,21 @@ package di
 
 import app.cash.sqldelight.driver.android.AndroidSqliteDriver
 import db.chargehub.ChargeHubDb
+import db.database.car.CarDatabase
+import db.database.car.CarDatabaseWrapper
 import db.database.user.UserDatabase
 import db.database.user.UserDatabaseWrapper
 import org.koin.dsl.module
 
 actual fun platformModules() = module {
+
     single {
         val driver = AndroidSqliteDriver(ChargeHubDb.Schema, get(), "chargehubdb.db")
         UserDatabaseWrapper(UserDatabase(driver))
     }
+
+    single {
```

Comment:
Aren't you creating multiple AndroidSqliteDriver instances? I'm not quite sure how this works but could it be that you only have to add `CarDatabaseWrapper(CarDatabase(driver))` with the driver above?

Reply:
The issue I had with this is that "AndroidSqliteDriver" requires "Context" as a parameter, the "get()" method seems to only be available inside the "single" scope and not directly inside the "module" scope, Im not sure how else we can access the context to pass it in.

Reply:
I See you changed it to extract the driver in the other pr's. Looks better!